arbor/util/pimpl_src.hpp

```diff
@@ -14,7 +14,7 @@ template<typename T>
 pimpl<T>::~pimpl() = default;
 
 template<typename T>
-pimpl<T>::pimpl() noexcept = default;
+pimpl<T>::pimpl() noexcept {}
```

Comment:
The usual wisdom is to use `default` over `{}`. What motivated this change?

Reply:
hipcc didn't like it if I recall correctly

Reply:
Shame on it then. Ok, but if you have the bandwidth to spare, could you annotate accordingly?

Reply:
done